src/shared/components/event-meta/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from 'react';
+import styles from './style.css';
+
+import TagsList from '../tags-list';
+import EventLinksList from '../event-links-list';
+
+const EventMeta = ({
+  event,
+}) => (
+  <div>
+  {
+    event.externalLinks || event.internalLinks ?
```

Comment:
Yeah probably. There's also potential for refactoring in a way of only passing down elements of the `event` and not the whole `event`.